src/Symfony/Component/Notifier/Bridge/Clickatell/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/clickatell-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony Clickatell Notifier Bridge",
+    "keywords": ["clickatell", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
```

Comment:
You should also add yourself as an author here

Reply:
Yes good, thank you !